util/args.py

```diff
@@ -51,7 +51,7 @@ def get_args() -> argparse.Namespace:
     parser.add_argument(
         "--optimizer",
         type=str,
-        default="AdamW",
+        default="Adam",

```

Comment:
Did it make a difference? Is it because you're using explicit weight decay now?

Reply:
Good question, intuitively I'd guess that it wouldn't matter much, but I don't know. For now I've just tried to make all the defaults match what the research paper mentions, as I presume those would correspond to generally reasonable values.